### ISSUE TITLE
[Bug]: Auto/Scheduled Backups may not be executing

### ISSUE THREAD

**Issue by raviwarrier** [advplyr/audiobookshelf#1590]
### Describe the issue

I have set my backups to run every 2 days, but from the image below you can see that the last auto backup happened on 6th Feb, instead of today (7th March on 0:00 hours).

![image](https://user-images.githubusercontent.com/42061703/223482094-4a31bca0-3883-414a-99ba-50c37cacd740.png)

The log shows removing a backup of 4th Feb (due to max backups to keep limitation), but none since 6th Feb 2023.

![image](https://user-images.githubusercontent.com/42061703/223483047-abd019bb-a3f3-48d1-9307-820351e6f4c6.png)

I don't know what's causing this issue as the logs don't show any errors in the past month. Let me know if there's any data that I can provide to help you debug this.

### Steps to reproduce the issue

1. NA

### Audiobookshelf version

2.2.16

### How are you running audiobookshelf?

Docker

**Comment by raviwarrier** [comment]
I checked the log from a little earlier and found this:

![image](https://user-images.githubusercontent.com/42061703/223483797-8b325002-a085-4adc-ace4-66b4d4a1441a.png)

This shows that when my RPI booted in the morning (around 4a), it found a bad backup archive. But that is also from 8th Feb 2023 and nothing else since then. Will check to see if the 9th March backup that is scheduled happens or not and report.

**Comment by raviwarrier** [comment]
![image](https://user-images.githubusercontent.com/42061703/223484516-607a2811-c6e8-456c-9b1c-fb598aa4f0fa.png)

Scratch that. It seems to be working... but only, now it seems, that ABS backup page is not displaying the latest backups and is stuck at 6th Feb 2023 backup.

**Comment by advplyr** [comment]
I think there was a version that was producing invalid backups and if they fail to load then they aren't counted as backups. Maybe try clearing out those old ones manually and we can see if we can reproduce this with fresh backups.

**Comment by raviwarrier** [comment]
Yeah... I just deleted all the old backup except the last two. Will check on 9th March and report back.

**Comment by raviwarrier** [comment]
Looks like everything is working fine. But I think the dates might be off... If you see in the OP, the next backup was scheduled to be at 09-March-2023 0:00, but...

![image](https://user-images.githubusercontent.com/42061703/224002441-bbab91b8-20bb-44a5-a7ac-d3562197f51c.png)

It happened at 08-March-2023 0:00. I don't know which one is wrong, but there is definitely a mismatch... 

Also, "0 0 */2 * *" means every 2nd day, so, why is it showing the schedule for every 3rd day. 

Last backup was on 6th, but it showed next backup on 9th (see image in OP)
This backup was on 8th, but it's showing next backup on 11th.

Should I open another issue for this?

**Comment by raviwarrier** [comment]
No. I don't think so. My backups have been working fine for a while now. You can close this. :)
